components/home/index.js

```diff
@@ -32,17 +34,13 @@ class Home extends Component {
       },
       interiorMode: false,
       nearbyMarkers: [],
-      isVisible: false,
       // eslint-disable-next-line react/no-unused-state
       isSearchVisible: true,
-      isGoVisible: false,
-      isSwitchAvailableIndestination: true,
-
       // current Concordia a user is in
```

Comment:
please be more specific. What is Concordia ?

Reply:
Done